lib/slanger/connection.rb

```diff
@@ -25,7 +25,7 @@ def error e
     end
 
     def establish
-      @socket_id = SecureRandom.uuid
+      @socket_id = "%d.%d" % [Process.pid, SecureRandom.random_number(10 ** 6)]
```

Comment:
Thanks to the pusher guys, cc @zimbatm it follows their format. It doesn't make a difference to us, but we can use the same validation method and it is probably less confusing if any switches between slanger/pusher